CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v36.0.1...main)
 
+* General
+  * **Breaking Change**: "deletion-request" pings now include the reason upload was disabled: `at_init` (Glean detected a change between runs) or `set_upload_enabled` (Glean was told of a change as it happened). ([#1593](https://github.com/mozilla/glean/pull/1593)).
```

Comment:
Hm, it's not really a breaking change though. No impact to users, just an enhancement of data delivered.

Reply:
It definitely is a breaking change to anyone who expects the reason to be empty. I'm thinking burnham.